.github/workflows/project-automation.yml

```
@@ -0,0 +1,138 @@
+    runs-on: ubuntu-latest
+    steps:
+      - name: Update Project Status
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.PROJECT_TOKEN }}
+          script: |
+            const { owner, repo } = context.repo;
+            const pr = context.payload.pull_request;
+            
+            const PROJECT_ID = 3;  // From your shared URL
+            
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)